client/src/modules/Admin/Components/Admin.tsx

```diff
@@ -22,11 +22,19 @@ export const Admin = () => {
   const [reportedReviews, setReportedReviews] = useState<Review[]>([])
   const [disableInit, setDisableInit] = useState<boolean>(false)
   const [disableNewSem, setDisableNewSem] = useState<boolean>(false)
+  const [disableUpdateProfs, setDisableUpdateProfs] = useState<boolean>(false)
```

Comment:
What about a universal disabled state? If this 1 state is true, then we simply set all the buttons to be disabled and locked.